server/pbench/bin/index-pbench

```diff
@@ -838,7 +870,76 @@ class ToolData(object):
     def _make_source_individual(self):
         """Read .csv files individually, emitting records for each row and
         column coordinate."""
-        pass
+        for csv in self.csv_files:
+            assert csv['header'][0] == 'timestamp_ms'
+            header = csv['header']
+            handler = csv['handler']
+            reader = csv['reader']
+            first_ts = None
+
+            while True:
```

Comment:
The "unify" method has to read from all `.csv` files at once, so it needs to use the `next()` function on each `.csv` file generator, the `reader`.

Here we don't have to, because we are processing the entire `.csv` file before moving on to the next one.  So I think we can replace the
```
while True:
    try:
        row = next(reader)
    except StopIteration:
        # ...
        break
```

loop with:

```
for row in reader:
```

Reply:
I started from the `_make_source_unified(self)` method and didn't know we could use for. Good catch and I will change this.